coffee-model/coffee-model-base/src/main/java/hu/icellmobilsoft/coffee/model/base/javatime/listener/TimestampsProvider.java

```diff
@@ -161,4 +164,13 @@ private boolean isInstantClass(Class<?> field) {
         return Instant.class.isAssignableFrom(field);
     }
 
+    @PostConstruct
+    private void init() {
+        String zoneIdEnv = System.getenv("TIMEZONE_ID");
```

Comment:
valahogy inkább Microprofile Configon keresztül kellene, meg a kulcs ne ide legyen beégetve magic string-ként, hanem egy kulcs gyűjtőben lenne a helye

Reply:
Azért nem ment microprofileba, mert akkor köteleznénk, hogy azt használják és arra gondoltam, hogy így általános ENV-ként mindenki számára megadható, nem biztos, hogy valid a meglátásom :). A kulcsot közben javítom igazad van.

Reply:
A model-nek letisztultnak kéne lennie minden ilyentől szerintem. Itt nem jó se a coffee-cdi se pedig a mp config bevezetése, mert azokkal további függőséget húznánk be, amit nagyon minimumon kéne tartani itt is mint ahogy a dto-knál.